examples/community/latent_consistency_img2img.py

```diff
@@ -513,9 +513,7 @@ class LCMSchedulerWithTimestamp(SchedulerMixin, ConfigMixin):
             there is no previous alpha. When this option is `True` the previous alpha product is fixed to `1`,
             otherwise it uses the alpha value at step 0.
         steps_offset (`int`, defaults to 0):
-            An offset added to the inference steps. You can use a combination of `offset=1` and
-            `set_alpha_to_one=False` to make the last step use step 0 for the previous alpha product like in Stable
-            Diffusion.
+            An offset added to the inference steps.
```

Comment:
```suggestion
            An offset added to the inference steps, as required by some model families.
```

How about something like this? Would it still be confusing?

Reply:
I think it's fair. Curious what @yiyixuxu or @sayakpaul think.

Reply:
ok let's do this?

Reply:
Updated.